src/gracetabnote.js

```diff
@@ -0,0 +1,32 @@
+// [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
+// @author Balazs Forian-Szabo
+//
+// ## Description
+//
+// A basic implementation of grace notes
+// to be rendered on a tab stave.
+//
+// See `tests/gracetabnote_tests.js` for usage examples.
+
+import { TabNote } from './tabnote';
+import { Flow } from './tables';
+
+export class GraceTabNote extends TabNote {
+  static get CATEGORY() { return 'gracetabnotes'; }
+  static get SCALE() { return 0.66; }
```

Comment:
So I think we could simplify this by moving `SCALE` to `this.render_options.scale`, and setting it in the constructor.